### PR TITLE
adding roll up page theme colors

### PR DESCRIPTION
**Pull request by souperneon**
cc @kkowalsky

### CODE REVIEW

File: src/stylesheets/project_specific/styleguidepage/styleguidepage.scss [hunk 1]
```diff
@@ -227,3 +227,42 @@ pre[class*="language-"] {
 .mz-lightgray {
   background-color: $mz-lightgray;
 }
+
+.mz-forestgreen-1 {
+  background-color: #366026;
```
Comment (rfriberg) [inline src/stylesheets/project_specific/styleguidepage/styleguidepage.scss]: Can you replace the hex values with variables?  It'll be easier for updating later, if needed.

e.g.:
```
.mz-forestgreen-1 {
    background-color: $mz-forestgreen-1;
}
```

**Comment by rfriberg** [comment]
Thanks @souperneon!  I'm excited to have these extra colors available.

**Review by rfriberg — CHANGES_REQUESTED** [review]

File: src/stylesheets/project_specific/styleguidepage/styleguidepage.scss (hunk 1 above)
Comment (souperneon) [reply src/stylesheets/project_specific/styleguidepage/styleguidepage.scss]: Doh!

**Review by rfriberg — APPROVED** [review]
:chefs_kiss:

**Pull request merged**